packages/react/src/components/Modal/index.mdx

```diff
@@ -0,0 +1,256 @@
+</Canvas>
+
+## Pre Built Modals
+
+As we know that developers love to have handy shortcuts for common use cases, we provide some pre built modals that we
+prevent you from adding `<Modal/>` and using `useModal()` each time you want to use those pre built modals.
+
+The way you will be able to use those pre built modals is by using async calls that return the decision of the user.
+
+> For instance the following code shows how to display a confirmation modal asking the user if he wants to continue or not.
+
+<Source
+  language='tsx'
+  dark
+  format={false}
+  code={`
+import { CunninghamProvider, Button, Modal } from "@openfun/cunningham-react";
+
+const App = () => {
+  const modals = useModals();
+  const ask = async () => {
+      const decision = await modals.confirmationModal();
+      alert("You decided: " + decision);
+  };
+  return <Button onClick={ask}>Open</Button>;
+};
+`}/>
+
+**About the `decision` variable:**
+
+- `decision` is truthy if the user accepted.
+- `decision` is null if the user denied.
+- `decision` is undefined if the user closed the modal via `esc` or close button.
+
+**Of course you can customize the title and the content of the modal by using the `props` first argument of the functions.**
+
+### Confirmation modal
+
+<Canvas>
+  <Story of={PreBuiltStories.ConfirmationModal} inline={false}/>
+</Canvas>
+
+### Delete confirmation modal
+
+<Canvas>
+  <Story of={PreBuiltStories.DeleteConfirmationModal} inline={false}/>
+</Canvas>
+
+### Message modal
+
+This modal can be used to display a message to the user. It is possible to use multiple variant of it by using `messageType` with `VariantType` enum.
+
+#### Neutral
+
+<Canvas>
+  <Story of={PreBuiltStories.NeutralModal} inline={false}/>
+</Canvas>
+<Canvas>
+  <Story of={PreBuiltStories.SuccessModal} inline={false}/>
+</Canvas>
+<Canvas>
+  <Story of={PreBuiltStories.InfoModal} inline={false}/>
+</Canvas>
+<Canvas>
+  <Story of={PreBuiltStories.ErrorModal} inline={false}/>
+</Canvas>
+<Canvas>
+  <Story of={PreBuiltStories.WarningModal} inline={false}/>
+</Canvas>
+
+## Props
+
+These are the props of `Modal`.
+
+<ArgTypes of={Modal} />
+
+## Design tokens
+
+Here a the custom design tokens defined by the Toast.
+
+| Token        	| Description                 	|
+|---------------	|-----------------------------	|
+| background-color 	| Default background color 	|
+| border-radius 	| Border radius of the modal 	|
+| border-color 	| Border color of the modal 	|
+| box-shadow 	| Box shadow of the modal 	|
+| title-font-weight 	| Font weight of the modal title	|
+| color 	| Color of the modal title 	|
+| content-font-size 	| Font size of the modal content	|
+| content-font-weight 	| Font weight of the modal content	|
+| content-color 	| Font Color of the modal content	|
+| small-width 	| Width of the small modal size	|
+| medium-width 	| Width of the medium modal size	|
+| large-width 	| Width of the large modal size	|
```

Comment:
You updated the token but you forgot to update this part of the doc accordingly.